bundles/binding/org.openhab.binding.plex/src/main/java/org/openhab/binding/plex/internal/PlexConnection.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.plex.internal;
+
+/**
+* Holds the connection properties for a Plex server 
+* 
+* @author Jeroen Idserda
+* @since 1.7.0
+*/
+public class PlexConnection {
```

Comment:
i'd suggest to rename this class into PlexConnectionProperties or ~Configuration since it does not represent the Connection by rather its Properties